sql_joins/sql_joins.md

```diff
@@ -1,14 +1,31 @@
 <!--
 author:   Joy Payton
 email:    paytonk@chop.edu
-version: 1.0.1
-module_template_version: 3.0.0
+version: 1.1.0
+current_version_description: Typo fix; update metadata
+module_type: standard
+docs_version: 2.0.0
 language: en
 narrator: US English Female
+mode: Textbook
 title: SQL Joins
 comment: Learn about SQL joins: what they accomplish, and how to write them.
 long_description: Usually, data in a SQL database is organized into multiple interrelated tables.  This means you will often have to bring data together from two or more tables into a single dataset to answer your research questions.  This "join" action is accomplished using `JOIN` commands.  This module teaches types of joins, join criteria, and how to write `JOIN` code.
-estimated_time: 1 hour
+estimated_time_in_minutes: 60
+good_first_module: false
+coding_required: true 
+coding_language: SQL
+coding_level: intermediate
+sequence_name: sql
```

Comment:
```suggestion
sequence_name: sql
previous_sequential_module: sql_intermediate
```
Or should database_normalization be the previous sequential module? I'm not sure if database_normalization actually fits into the sql sequence.  This is maybe a little tricky, but it should definitely be one of those as the previous sequential module.